modules/jooby-avaje-inject/pom.xml

```diff
@@ -11,7 +11,11 @@
 
   <modelVersion>4.0.0</modelVersion>
   <artifactId>jooby-avaje-inject</artifactId>
-
+  
+  <properties>
+    <maven.compiler.proc>full</maven.compiler.proc>
```

Comment:
@SentryMan maven/javac compiler fails with `-proc:full` on JDK 17.

     Fatal error compiling: error: invalid flag: -proc:full 

Any idea of why?

Reply:
What JDK 17 version do you have, they back ported this feature from 21 a while back

Reply:
I see.

```
openjdk version "17.0.10" 2024-01-16
OpenJDK Runtime Environment Temurin-17.0.10+7 (build 17.0.10+7)
OpenJDK 64-Bit Server VM Temurin-17.0.10+7 (build 17.0.10+7, mixed mode)
```

Will update my sdk.

Reply:
Fixed. Thanks.